koku/api/provider/serializers.py

```diff
@@ -346,8 +347,9 @@ def update(self, instance, validated_data):
         provider_type = Provider.PROVIDER_CASE_MAPPING.get(provider_type)
         validated_data["type"] = provider_type
         if instance.type != provider_type:
-            error = {"Error": "The Provider Type cannot be changed with a PUT request."}
-            raise serializers.ValidationError(error)
+            key = "Error"
```

Comment:
I'm pretty sure this is a left over from when we supported providers and sources.

Reply:
I'll remove this then...